scalding-repl/src/main/scala/com/twitter/scalding/ScaldingILoop.scala

```diff
@@ -18,12 +18,16 @@ package com.twitter.scalding
 import java.io.File
 
 import scala.tools.nsc.interpreter.IR
+import scala.tools.nsc.GenericRunnerSettings
 
 /**
  * A class providing Scalding specific commands for inclusion in the Scalding REPL.
  */
 class ScaldingILoop
   extends ILoopCompat {
+
+  settings = new GenericRunnerSettings({ s => echo(s) })
```

Comment:
To instantiate the settings, I had to provide an `errorFn :: String -> Unit`.
Does this seem like a suitable error function? I could not copy the error function from the default settings because, by default, ILoop's `settings` is null.